opendistro-elasticsearch-job-scheduler.release-notes

```diff
@@ -1,3 +1,8 @@
+## version 0.10.0
+
+### Notable Changes
+* Feature [PR #18](https://github.com/opendistro-for-elasticsearch/job-scheduler/pull/18) Add Support for Elasticsearch 6.8.1
+
 ## 2019-04-10
```

Comment:
Just noticed that the original release just has a date with no version, might be worth including the initial version, like we did with Alerting's